### PR TITLE
[16.0] [MIG] product_sequence

### PR DESCRIPTION
**Pull request by marielejeune**

### CODE REVIEW

**Comment by marielejeune** [comment]
Depends on https://github.com/OCA/product-attribute/pull/1159

File: product_sequence/models/ir_sequence.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+from odoo import api, models
+
+
+class IrSequence(models.Model):
+    _inherit = "ir.sequence"
+
+    @api.model
+    def get_category_sequence_id(self, category=False):
+        if self.env.user.company_id.use_parent_categories_to_determine_prefix:
+            while not category.sequence_id and category.parent_id:
```
Comment (apineux) [inline product_sequence/models/ir_sequence.py]: This code is not robust as category can be False (default value). You should add the condition "while category and ..." OR you should remove the default value of category to make sure that this parameter is set.

**Review by apineux — CHANGES_REQUESTED** [review]

File: product_sequence/models/ir_sequence.py (hunk 1 above)
Comment (marielejeune) [reply product_sequence/models/ir_sequence.py]: Done

**Review by lmignon — APPROVED** [review]
LGTM (Code review)

**Comment by lmignon** [comment]
@marielejeune Can you rebase your branch? #1159 is now merged.

**Review by lmignon — APPROVED** [review]
Thank you @marielejeune  :+1:  (Code review + Functional test)

**Comment by lmignon** [comment]
ping @rousseldenis

**Comment by rousseldenis** [comment]
@apineux Could you update your review ?

**Review by rousseldenis — APPROVED** [review]

**Comment by rousseldenis** [comment]
Merging as comments have been attended

/ocabot merge patch

**Pull request merged**